index.js

```diff
@@ -66,6 +66,8 @@ function handleOpts(opts) {
 		opts._config.baseConfig = 'xo/esnext';
 	}
 
+	opts._config.plugins.push('no-use-extend-native');
```

Comment:
Did the easy thing and placed it here, but would you like me to set it in `DEFAULT_CONFIG` instead?